scripts/xtensa-build-zephyr.py

```diff
@@ -1173,7 +1195,7 @@ def gzip_compress(fname, gzdst=None):
 
 RI_INFO_UNSUPPORTED += ['imx8', 'imx8x', 'imx8m', 'imx8ulp', 'imx95']
 RI_INFO_UNSUPPORTED += ['rn', 'acp_6_0']
-RI_INFO_UNSUPPORTED += ['mt8186', 'mt8195']
+RI_INFO_UNSUPPORTED += ['mt8186', 'mt8188', 'mt8195', 'mt8196']
```

Comment:
Not confident to say I understand this script but these versions should be supported after this PR?

Reply:
Yes.  Though this particular line is just flagging that these platforms lack the Intel-specific CSE manifest that the sof_ri_info.py script expects, so it shouldn't be run.   The list of buildable platforms is above here, in the extra_platform_configs and platform_configs_all arrays.